dialogue-annotations-processor/src/main/java/com/palantir/dialogue/annotations/processor/data/EndpointDefinition.java

```diff
@@ -34,4 +34,9 @@
     List<ArgumentDefinition> arguments();
 
     TypeName returns();
+
+    @Value.Derived
+    default String channelFieldName() {
+        return endpointName().get() + "Channel";
```

Comment:
I rather like this way of solving the "I need this value in a bunch of generators" rather than static util methods scattered around everywhere.

Reply:
It's what they recommend in "Effective Java" these days :-)